pkg/debian/salt-common.preinst

```diff
@@ -31,11 +31,5 @@ case "$1" in
             -s $SALT_SHELL  \
             -g $SALT_GROUP  \
              $SALT_USER
-    # 5. adjust file and directory permissions
```

Comment:
Hi @barneysowood, are you able to elaborate on why you are removing this piece?

Reply:
Hi @MKLeb - because the `salt` owner that the salt-master process runs as shouldn't be able to write to /opt/saltstack/salt - this would allow it modify the whole salt install...